vm/policy.go

```diff
@@ -1,212 +0,0 @@
-		resp, err := client.HostPolicy(context.Background(), &req)
-		if err != nil {
-			return fmt.Errorf("failed to send policy")
-		}
-		fmt.Printf("Policy %s \n", resp.Status)
-		return nil
-
-	}
-	resp, err := client.ContainerPolicy(context.Background(), &req)
-	if err != nil {
-		return fmt.Errorf("failed to send policy")
-	}
-	fmt.Printf("Policy %s \n", resp.Status)
-	return nil
-
-}
-
-func sendPolicyOverHTTP(address string, kind string, policyEventData []byte) error {
-
-	timeout := time.Duration(5 * time.Second)
-	client := http.Client{
-		Timeout: timeout,
-	}
-
-	var url string
-	if kind == KubeArmorHostPolicy {
-		url = address + "/policy/kubearmor"
-	} else {
-		url = address + "/policy/cilium"
-	}
-
-	request, err := http.NewRequest("POST", url, bytes.NewBuffer(policyEventData))
-	request.Header.Set("Content-type", "application/json")
-	if err != nil {
-		return fmt.Errorf("failed to send policy")
-	}
-
-	resp, err := client.Do(request)
-	if err != nil {
-		return fmt.Errorf("failed to send policy")
-	}
-	defer func() {
-		if err := resp.Body.Close(); err != nil {
-			kg.Warnf("Error closing http stream %s\n", err)
-		}
-	}()
-
-	fmt.Println("Success")
-	return nil
-}
-
-// PolicyHandling Function recives path to YAML file with the type of event and emits an Host Policy Event to KubeArmor gRPC/HTTP Server
-func PolicyHandling(t string, path string, o PolicyOptions, httpAddress string, isKvmsEnv bool) error {
-	var k struct {
-		Kind string `json:"kind"`
-	}
-
-	policyFile, err := os.ReadFile(filepath.Clean(path))
-	if err != nil {
-		return err
-	}
-
-	policies := strings.Split(string(policyFile), "---")
-
-	for _, policy := range policies {
-
-		if matched, _ := regexp.MatchString("^\\s*$", policy); matched {
-			continue
-		}
-
-		js, err := yaml.YAMLToJSON([]byte(policy))
-		if err != nil {
-			return err
-		}
-
-		err = json.Unmarshal(js, &k)
-		if err != nil {
-			return err
-		}
-
-		var containerPolicy tp.K8sKubeArmorPolicy
-		var hostPolicy tp.K8sKubeArmorHostPolicy
-		var networkPolicy v2.CiliumNetworkPolicy
-		var policyEvent interface{}
-
-		if k.Kind == KubeArmorHostPolicy {
-			err = json.Unmarshal(js, &hostPolicy)
-			if err != nil {
-				return err
-			}
-
-			policyEvent = tp.K8sKubeArmorHostPolicyEvent{
-				Type:   t,
-				Object: hostPolicy,
-			}
-
-		} else if k.Kind == KubeArmorPolicy {
-			err = json.Unmarshal(js, &containerPolicy)
-			if err != nil {
-				return err
-			}
-
-			policyEvent = tp.K8sKubeArmorPolicyEvent{
-				Type:   t,
-				Object: containerPolicy,
-			}
-
-		} else if k.Kind == CiliumNetworkPolicy || k.Kind == CiliumClusterwideNetworkPolicy {
-			err = json.Unmarshal(js, &networkPolicy)
-			if err != nil {
-				return err
-			}
-
-			if networkPolicy.Spec == nil {
-				continue
-			}
-
-			policyEvent = NetworkPolicyRequest{
-				Type:   t,
-				Object: networkPolicy,
-			}
-
-		}
-
-		policyEventData, err := json.Marshal(policyEvent)
-		if err != nil {
-			return err
-		}
-
-		if isKvmsEnv {
```

Comment:
We only need to remove the code where ks KVMs env. Rest of the policy add and delete is for KubeArmor unorchestrated host and containers mode.

Reply:
got it. complete missed that part. thanks for noticing. :+1: